### PR TITLE
Add dense and sparse tiny YOLO v2 models trained from scratch by AVA.

### PR DESCRIPTION
**Pull request by eldercrow**
3 tiny YOLO v2 models are added, with (0%, 30%, 60%) weight sparsities.

### CODE REVIEW

**Comment by eizamaliev** [comment]
@eldercrow could these models run with [YOLO detection demo](https://github.com/opencv/open_model_zoo/tree/develop/demos/object_detection_demo_yolov3_async) or its [Python analogue](https://github.com/opencv/open_model_zoo/tree/develop/demos/python_demos/object_detection_demo_yolov3_async)

**Comment by eldercrow** [comment]
yes it should be

2020년 1월 26일 (일) 23:41, eizamaliev <notifications@github.com>님이 작성:

**Comment by eldercrow** [comment]
@vladimir-dudnik I've added links, could you please check them?

File: models/intel/yolo-tiny-v2-ava-0001/description/yolo-tiny-v2-ava-0001.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+## Example
+
+## Specification
+
+| Metric                          | Value                                     |
+|---------------------------------|-------------------------------------------|
+| Mean Average Precision (mAP)    | 35.37%                                    |
+| Flops                           | 6.97Bn*                                   |
+| Source framework                | Tensorflow**                              |
```
Comment (IRDonch) [inline models/intel/yolo-tiny-v2-ava-0001/description/yolo-tiny-v2-ava-0001.md]:
```suggestion
| Source framework                | TensorFlow**                              |
```

File: models/intel/yolo-v2-ava-sparse-35-0001/description/yolo-tiny-v2-ava-sparse-30-0001.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
+# yolo-v2-ava-sparse-35-0001
```
Comment (IRDonch) [inline models/intel/yolo-v2-ava-sparse-35-0001/description/yolo-tiny-v2-ava-sparse-30-0001.md]: The filename doesn't match the model name.

File: tools/accuracy_checker/configs/yolo-tiny-v2-ava-0001.yml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    launchers:
+      - framework: dlsdk
+        tags:
+          - FP32
+        model: intel/yolo-tiny-v2-ava-0001/FP32/yolo-tiny-v2-ava-0001.xml
+        weights: intel/yolo-tiny-v2-ava-0001/FP32/yolo-tiny-v2-ava-0001.bin
+        adapter:
+          type: yolo_v2
+          anchors: tiny_yolo_v2
+      - framework: dlsdk
```
Comment (eaidova) [inline tools/accuracy_checker/configs/yolo-tiny-v2-ava-0001.yml]: please also add launcher for FP16 for all 3 models

File: models/intel/yolo-v2-ava-sparse-35-0001/description/yolo-tiny-v2-ava-sparse-30-0001.md (hunk 2 above)
Comment (vladimir-dudnik) [reply models/intel/yolo-v2-ava-sparse-35-0001/description/yolo-tiny-v2-ava-sparse-30-0001.md]: seems fixed

File: tools/accuracy_checker/configs/yolo-tiny-v2-ava-0001.yml (hunk 3 above)
Comment (vladimir-dudnik) [reply tools/accuracy_checker/configs/yolo-tiny-v2-ava-0001.yml]: fixed

**Comment by IRDonch** [comment]
Jenkins: test this please

**Comment by IRDonch** [comment]
OK, I'm not sure what went wrong with the history, but a quick rebase fixed it.

**Comment by eaidova** [comment]
@eldercrow  please do not merge develop branch to your PR. it makes your commit history messy. Please rebase it on upstream instead.

@vladimir-dudnik, AVA ICNet configs also were added in this PR, as I can see. So you need to decide what we should to do with https://github.com/opencv/open_model_zoo/pull/908

**Comment by IRDonch** [comment]
Jenkins: test this please

**Comment by IRDonch** [comment]
Jenkins please retry a build

**Comment by PrashantDandriyal** [comment]
Hi,
Where are the sparse models? All I can see is empty directories just with README.md files... 
Regards

**Comment by IRDonch** [comment]
@PrashantDandriyal The models themselves are added separately. Pre-release versions of these models have already been merged to the `develop` branch.

**Pull request merged**